test/jdk/com/sun/management/ThreadMXBean/ThreadAllocatedMemoryArray.java

```diff
@@ -118,18 +116,8 @@ public static void main(String[] argv)
         goSleep(400);
 
         long[] sizes1 = mbean.getThreadAllocatedBytes(ids);
-
         for (int i = 0; i < NUM_THREADS; i++) {
-            long newSize = sizes1[i];
-            if (sizes[i] > newSize) {
-                throw new RuntimeException("TEST FAILED: " +
-                    threads[i].getName() +
-                    " previous allocated bytes = " + sizes[i] +
-                    " > current allocated bytes = " + newSize);
-            }
-            System.out.println(threads[i].getName() +
-                " Previous allocated bytes = " + sizes[i] +
-                " Current allocated bytes = " + newSize);
+            checkResult(threads[i], sizes[i], sizes1[i]);
```

Comment:
Since we are cleaning up the test anyway, can we / should we rename `sizes` -> `before`, `size1` -> `after`?